### ISSUE TITLE
Support only a single cube data set per cubeviz layout

### ISSUE THREAD

**Issue by drdavella** [spacetelescope/cubeviz#302]
Proposal: we should not allow users to load multiple cube data sets into the same cubeviz tab at the same time. I believe this is what caused the performance issues that Simona reported today, along with the quirkiness with layers and subsets that she demonstrated.

As far as I am aware, there should not be use cases where different 3D cube data sets need to be displayed in the same cubeviz tab at the same time. If this assumption is not correct, then we are going to need to give some careful consideration to how these cases should be handled.

**Comment by astrofrog** [comment]
I think that in addition to disallowing more than one dataset per tabl, in the documentation we should make it clear that users who want to try more advanced workflows and have multiple datasets in the same tab can use glue itself rather than cubeviz specifically - that is they can just create an arbitrary number of image and spectrum viewers in the same tab.

**Comment by kassin** [comment]
Is this part of #299 ?

**Comment by kassin** [comment]
Users will definitely want to open more than one cube at a time in CubeViz.  They will want to compare multiple cubes and do arithmetic with them, among other things.

**Comment by drdavella** [comment]
@kassin I can confirm that the issues that @smei100 reported today during the telecon were caused by having multiple cube data sets loaded at the same time. This is a use case we have not accounted for in our code, and it will take a non-trivial amount of effort to remedy. In the interim, we should at the very least caution users about this.

**Comment by smei100** [comment]
Thank you this is very useful to know

**Comment by kassin** [comment]
@drdavella Agreed.  Thank you for confirming.  And thank you @smei100 for finding this bug.
